docs/operators/operator-node/node-setup/best-practices.md

```diff
@@ -0,0 +1,224 @@
+---
+sidebar_label: 'Best Practices'
+sidebar_position: 3
+---
+
+# Best Practices for Running SSV Node
+
+This guide outlines key recommendations to ensure the best Performance and Correctness of your node. The guide will be updated as we get new information.
+
+The page is divided into three parts:
+- [**Introduction**](#introduction) - understanding the Performance, Correctness, and their key factors.
+- [**Major impact**](#major-impact) - vital tips, more obvious.
+- [**Minor impact**](#minor-impact) - fine-tuning tips, nice-to-have, less obvious.
+
+
+## **Introduction**
+
+Performance you see on [SSV Explorer](https://explorer.ssv.network/) is essentially participation rate - how many duties your node participated in. Correctness refers to how timely and accurate duties are, and it affects validators' rewards.
+
+Several key factors influence this:
+- **Setup Configuration:** Software used to manage nodes, configured ports and the firewall help eliminate delays.
+- **Network Throughput & Latency:** Minimal network delay is critical, especially for production setups.
+- **Hardware Resources:** Adequate CPU, sufficient RAM, and fast, reliable storage are necessary.
+
+## **Major impact**
+More obvious, more impactful.
```

Comment:
"more obvious, more impactful"...peanuts? stones?

I know it's kinda obvious, but make this a one-liner description...